app/src/main/java/com/wesleyfranks/todo24/ui/completed/CompletedViewModel.kt

```diff
@@ -1,13 +1,45 @@
 package com.wesleyfranks.todo24.ui.completed
 
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
+import android.content.Context
+import androidx.lifecycle.*
+import com.wesleyfranks.todo24.data.Todo
+import com.wesleyfranks.todo24.data.TodoRepository
+import kotlinx.coroutines.launch
 
 class CompletedViewModel : ViewModel() {
 
-    private val _text = MutableLiveData<String>().apply {
-        value = "This is completed Fragment"
+    companion object{
```

Comment:
Everything in `CompletedViewModel` seems out of scope of this PR, as you can't see the completed items here. Pull requests should be limited in scope, and fully functional.